src/mscorlib/System.Private.CoreLib.csproj

```diff
@@ -34,6 +34,7 @@
     <SignAssembly>true</SignAssembly>
     <DelaySign>true</DelaySign>
     <DefineConstants>$(DefineConstants);CORECLR;_USE_NLS_PLUS_TABLE;RESOURCE_SATELLITE_CONFIG;CODE_ANALYSIS_BASELINE</DefineConstants>
+    <DefineConstants Condition="'$(TargetGroup)'=='netcoreapp' OR '$(TargetGroup)'=='uap' OR '$(TargetGroup)'=='uapaot'">$(DefineConstants);netcoreapp</DefineConstants>
```

Comment:
This can defined unconditionally here. The condition is not necessary.

Reply:
Done!